lib/loader/index.js

```diff
@@ -173,15 +141,15 @@ function postInstantiate(baseModule, instance) {
   function getView(align, signed, float) {
     if (float) {
       switch (align) {
-        case 2: return F32;
-        case 3: return F64;
+        case 2: return new Float32Array(memory.buffer);
```

Comment:
What about cache memory.buffer to `cost buf = memory.buffer`? It could be reduce code size of `getView`

Reply:
Hmm, not sure. In Wasm terms that'd be one local + a local.get + a local.set we don't need since the field is read exactly once per branch. Most likely negligible, so no strong preference.

Reply:
Could also do things like

```ts
  case 0: return new (signed ? Int8Array : Uint8Array)(memory.buffer);
```

but I don't know if that has overhead, like in Wasm terms, an indirect call.

Reply:
Also great! Js could optimize this things good as well